src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
+    private final Ball ball = new Ball();
+    private final ColorSupplier newColor = new ColorSupplier();
```

Comment:
The variable name 'newColor' is misleading as it suggests that it is a color, but it is actually a ColorSupplier. Consider renaming it to 'colorSupplier' to accurately reflect its purpose.